integrations/amazonKeys/cli/init.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+readonly IFS=$'\n\t'
+readonly ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SCRIPT_NAME="$( basename "$0" )"
+readonly ARGS=("$@")
```

Comment:
Turns out that if I just copy the value, it will be copied as a string. This makes an array copy.